plugin/trino-hive/src/main/java/io/trino/plugin/hive/SchemaAlreadyExistsException.java

```diff
@@ -25,12 +25,17 @@ public class SchemaAlreadyExistsException
 
     public SchemaAlreadyExistsException(String schemaName)
     {
-        this(schemaName, format("Schema already exists: '%s'", schemaName));
+        this(schemaName, format("Schema already exists: '%s'", schemaName), null);
```

Comment:
No need to declare the error message in two places:
```suggestion
        this(schemaName, null);
```